pkgs/tools/X11/autokey/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, python3Packages }:
```

Comment:
Could you please replace `callPackage` in `all-packages.nix` with `python3Packages.callPackage` and explicitly declare all inputs to the derivation in the function arguments? 

With that I mean replace the line `{ stdenv, fetchurl, python3Packages }:` with `{ stdenv, fetchurl, xlib, pyinotify, dbus-python, buildPythonApplication }:`.

Reply:
Thank you the package starts building but fails due to dbus-python not being found, which is why i had dbus-python in buildinputs but no sucess